workers/unity/Packages/com.improbable.gdk.mobile/Android/Utility/AndroidDeviceInfo.cs

```diff
@@ -0,0 +1,39 @@
+using UnityEngine;
+
+namespace Improbable.Gdk.Mobile.Android
+{
+    public static class AndroidDeviceInfo
+    {
+        public const string AndroidStudioEmulatorDefaultCallbackIp = "10.0.2.2";
+
+        private static bool isInitialised = false;
+        private static bool isEmulator = false;
+
+        public static bool IsAndroidStudioEmulator()
+        {
+            if (!isInitialised)
+            {
+                AndroidJavaObject build = new AndroidJavaObject("android.os.Build");
```

Comment:
Is it safe to dispose of this object? It's a disposable so I'm guessing it should be disposed

Reply:
could just use `using` to automatically dispose of it?